include/aikido/robot/ConcreteRobot.hpp

```diff
@@ -310,4 +354,6 @@ class ConcreteRobot : public Robot
 } // namespace robot
 } // namespace aikido
 
+#include "detail/ConcreteRobot-impl.hpp"
```

Comment:
Good! These functions are simple enough to not be a separate compilation unit.

Reply:
Got it!